templates/projects.hbs

```diff
@@ -1,8 +1,123 @@
 {{! to compile, inside of templates: handlebars projects.hbs -f projects.hbs.js }}
-<ul>
+{{!-- <ul>
   {{#current_projects}}
     <li>
       From Handlbars template: {{name}}
     </li>
   {{/current_projects}}
-</ul>
+</ul> --}}
+
+<p>
+  Below are a selection of a few projects the brigade has worked on.
```

Comment:
how about 
**"A few projects the brigade has worked on:"**
or
**"A curated list of the projects the brigade has worked on:"**